test/C/src/arduino/Blink.lf

```diff
@@ -0,0 +1,42 @@
+/** This example demonstrates a very simple use of protocol buffers
+ *  within a reactor. It encodes and decodes a very simple protocol
+ *  buffer definition in Person.proto. This reactor is heavily
+ *  based on the examples at https://github.com/protobuf-c/protobuf-c/wiki/Examples.
+ *  This example just packs and unpacks a message.
+ *
+ *  To run this example first install the protocol buffers compiler
+ *  from https://github.com/protocolbuffers/protobuf. It is also
+ *  available from homebrew on a Mac via
+ *
+ *       $ brew install protobuf
+ *
+ *  Building protobuf from source is slow, so avoid doing that
+ *  if possible. Next install the C plugin for protocol buffers from
+ *
+ *      https://github.com/protobuf-c/protobuf-c
+ *
+ *  The code generator assumes
+ *  that executables are installed within the PATH. On a Mac, this is
+ *  typically at /usr/local/bin.
+ */
```

Comment:
Is this comment up to date?

(Also the terminating newline is missing.)

Reply:
Sorry, forgot to push the change that updated the comment line. Should be fixed now.